### DISCUSSION TITLE
Data link to legend entries

### DISCUSSION THREAD

**Discussion by Coalado** [grafana/grafana#37845] [Dashboards, Panels & Visualizations]
I would like to customize the link/action of a legend entry. By default, a click hides/shows the related series. 

**Why is this needed**:
I'd like to open a new dashboard on click that shows details about the entry.  Using data-links is not convenient, because it is hard to select the correct series in the graph.

Maybe an optional [x] override legend action checkbox for regular data links would already solve the problem.

**Comment by ashharrison90** [comment]
hey @Coalado 👋 

as per https://github.com/grafana/grafana/issues/36683, we're moving feature requests to Github discussions until we're ready to commit to implementing the feature.

thanks for raising this! 🙌

**Reply by sandeshjagannath** [reply]
Hey @ashharrison90 
Is there a plan to implement this feature in the near future? One of our requirements also needs the ability to make the legends on our Grafana dashboard as links for the series..

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
